bears/general/LineBreakBear.py

```diff
@@ -0,0 +1,133 @@
+from queue import Queue
+
+from coalib.results.AbsolutePosition import AbsolutePosition
+from coalib.results.Diff import Diff
+from coalib.bearlib.languages.LanguageDefinition import LanguageDefinition
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result, RESULT_SEVERITY
+from coalib.results.SourceRange import SourceRange
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+
+from bears.general.AnnotationBear import AnnotationBear
+from bears.general.RangeHelpers import get_specified_block_range
```

Comment:
Only as long as that's not a problem with other functionalities. Like our generated online help and [bear docs](https://github.com/coala-analyzer/bear-docs/blob/master/README.rst#all). It's really inconsistent anyways. I guess if the "moving away commit" is part of this PR it is fine though